sdk/search/search-documents/README.md

```diff
@@ -25,30 +25,27 @@ Use the client library to:
 
 ## Getting started
 
-### Currently supported environments
-
-- [Node.js](https://nodejs.org/) version 8.x.x or higher
-
 ### Prerequisites
 
+- [Node.js](https://nodejs.org/) version 8.x.x or higher
 - An [Azure subscription][azure_sub].
 - An existing [Azure Cognitive Search][search_resource] resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
```

Comment:
we always lower-case "portal" in the docs (Azure portal)

Reply:
Done. Will be updated in the next commit